<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QDialog" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1448</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1501</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f57224; </string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(240, 240, 249, 0); 
color: black;            
border: none;              
border-radius: 5px;       
padding: 5px;   
cursor: pointer; </string>
    </property>
    <property name="text">
     <string>Hotel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(240, 240, 249, 0); 
cursor: pointer; 
color: black;            
border: none;              
border-radius: 5px;       
padding: 5px;   </string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(240, 240, 249, 0); 
color: black;            
border: none;              
border-radius: 5px;       
padding: 5px;   
cursor: pointer; </string>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>70</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(240, 240, 249, 0); 
color: black;            
border: none;              
border-radius: 5px;       
padding: 5px;   
cursor: pointer; </string>
    </property>
    <property name="text">
     <string>Cab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>0</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(240, 240, 249, 0); 
color: black;            
border: none;              
border-radius: 5px;       
padding: 5px;   
cursor: pointer; </string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(240, 240, 249, 0); 
color: black;            
border: none;              
border-radius: 5px;       
padding: 5px;   
cursor: pointer; </string>
    </property>
    <property name="text">
     <string>Guide</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>0</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(240, 240, 249, 0); 
color: black;            
border: none;              
border-radius: 5px;       
padding: 5px;   
cursor: pointer; </string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>1271</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F5F5F5; </string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;</string>
   </property>
   <property name="text">
    <string>Packages:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
